src/sagemaker_inference/default_inference_handler.py

```diff
@@ -85,4 +85,7 @@ def default_output_fn(self, prediction, accept):  # pylint: disable=no-self-use
             obj: prediction data.
 
         """
-        return encoder.encode(prediction, accept), accept
+        for content_type in utils.parse_accept(accept):
+            if content_type in encoder.SUPPORTED_CONTENT_TYPES:
+                return encoder.encode(prediction, content_type), content_type
+        return encoder.encode(prediction, content_types.JSON), content_types.JSON
```

Comment:
Also, not sure if `content_types.JSON` should be a named constant like `DEFAULT_CONTENT_TYPE`. If so, which module would I put it in?

Reply:
I think it'd make more sense to raise an error with having an unsupported content type - presumably if we're at this point in the code, JSON wasn't one of the supported content types

basically the same logic in `encode`: https://github.com/aws/sagemaker-inference-toolkit/blob/master/src/sagemaker_inference/encoder.py#L107